tests/tools/onert_train/src/args.cc

```diff
@@ -113,9 +113,9 @@ T checkValidation(const std::string &arg_name, const std::vector<T> &valid_args,
 
 // generate a help message based on the valid_args and default_arg
 template <typename T>
-std::string genHelpMsg(const std::string &arg_name, const std::vector<T> &valid_args, T default_arg)
+std::string genHelpMsg(const std::string &arg_name, const std::vector<T> &valid_args)
 {
-  std::string msg = arg_name + " (default: " + onert_train::to_string(default_arg) + ")\n";
+  std::string msg = arg_name + " (default: use model file parameter)\n";
```

Comment:
Some other candidate words to use here.

```suggestion
  std::string msg = arg_name + " (If nothing is given, model hyper param is selected)\n";
```

Reply:
@YongseopKim How about this word? Is it a bit better or not..?

Reply:
Great!